### ISSUE TITLE
Feature: nim web/object/kv do not support --json flag

### ISSUE THREAD

**Issue by rabbah** [nimbella/nimbella-cli#37]
I propose adding `--json` on the web/object/kv operations to consume (list) output as json.

**Comment by bhageena** [comment]
I have added the `--json` (`-j`) flag on the output of web, object, and kv [list] commands
the output comes like this

**nim [object|web] list  --json --long**
```json
{
  "files": [
    {
      "size": "1.3 KiB",
      "updated": "2021-01-24T05:52:13.854Z",
      "name": "README.md"
    },
    {
      "size": "1.3 KiB",
      "updated": "2021-01-24T07:07:01.686Z",
      "name": "test.md"
    }
  ]
}
```


**nim kv list --json**
```json
{
  "keys": [
    "a",
    "b"
  ]
}
```
@joshuaauerbachwatson @rabbah 
I am not sure about the consume part, till now via **[web|object] [add|update]** we are only uploading files that are already in the filesystem. Do we need to give functionality, so that given JSON content could be written as a file in web or object store with the given filename?   
Will update, in this case, augment the existing file content or replace it. 

I believe **kv set --json <jsonContent>** will extract key-value pairs out of given JSON, is that correct understanding, how will we do with nested JSON.
Every key-value pair will be independent or we need to make keys like `parent.child.child`

**Comment by joshuaauerbachwatson** [comment]
@bhageena , my understanding of the scope here is that we want to support JSON on the output side of the command, not the input side.   Since the issue was opened by @rabbah, though, I may hold off on merging PR #55 until he has commented.

> I believe kv set --json will extract key-value pairs out of given JSON

This was not pre-existing function and I don't changes to kv set in the PR, so I assume you are asking whether to add this.   Again, I believe it is beyond the scope and could be fairly complex, as you allude to.

**Comment by rabbah** [comment]
> my understanding of the scope here is that we want to support JSON on the output side of the command, 

that was my intent in opening this issue.

**Comment by bhageena** [comment]
sure, then I think we are good to go

**Comment by joshuaauerbachwatson** [comment]
This was fixed by PR #55 and is awaiting a stable version.

**Comment by joshuaauerbachwatson** [comment]
In stable version 1.12
